src/bindings/python/tests/__init__.py

```diff
@@ -28,36 +28,36 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
 xfail_issue_69444 = xfail_test(reason="ONNX Resize - AssertionError: Mismatched elements.")
 skip_issue_67415 = pytest.mark.skip(reason="RuntimeError: Unsupported data type for when filling blob!")
 xfail_issue_67415 = xfail_test(reason="RuntimeError: Unsupported data type for when filling blob!")
-xfail_issue_33488 = xfail_test(reason="RuntimeError: nGraph does not support the following ONNX operations: "
+xfail_issue_33488 = xfail_test(reason="RuntimeError: graph does not support the following ONNX operations: "
```

Comment:
```suggestion
xfail_issue_33488 = xfail_test(reason="RuntimeError: OV does not support the following ONNX operations: "
```
Maybe for all xfails?